views/module.pug

```diff
@@ -45,24 +39,38 @@ block content
           .form-group.col-sm
             label(for='shape') Shape and size
             select.form-control(id='shape' name='shape' required)
-              option !{mod.shape}
+              if mod.shape != ''
+                option !{mod.shape}
               option R3
               option R2.3
               option T3
               option T2.3
           .form-group.col-sm
             label(for='orientation') Angled Orientation
             select.form-control(id='orientation' name='orientation')
-              option !{mod.orientation}
+              if mod.orientation != ''
+                option !{mod.orientation}
```

Comment:
instead of adding an extra option, this should really just be setting the value on the `select` tag

Reply:
When I set the value it does not select the option, so it may have the value RH but will show the first option of flat instead